manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Nushell.Nushell
+PackageVersion: 0.34.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.34.0/nu_0_34_0_windows.msi
+  InstallerSha256: 76DB00921454EE055F417E19C44C890A7EA0775C84E1FD3C2E0232E0DAA31E38
+  ProductCode: '{B713991D-3589-4BD8-B55D-D8A14EC1A72E}'
+ManifestType: installer
+ManifestVersion: 1.0.0

```

Comment:
```suggestion
PackageIdentifier: Nushell.Nushell
PackageVersion: 0.34.0
MinimumOSVersion: 10.0.0.0
InstallModes:
- interactive
- silent
- silentWithProgress
Commands:
- nu
Installers:
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: msi
  Scope: machine
  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.34.0/nu_0_34_0_windows.msi
  InstallerSha256: 76DB00921454EE055F417E19C44C890A7EA0775C84E1FD3C2E0232E0DAA31E38
  UpgradeBehavior: install
  ProductCode: '{B713991D-3589-4BD8-B55D-D8A14EC1A72E}'
ManifestType: installer
ManifestVersion: 1.0.0
```

Reply:
Why is `MinimumOSVersion` required?

Reply:
Oh! it was autogenerated using `YamlCreate.ps1` script. I will remove it.